### ISSUE TITLE
starting balance is not null

### ISSUE THREAD

**Issue by SafetyIng** [powerpaul17/nc_money#50]
**Is your feature request related to a problem? Please describe.**
I am currently seeing the "Starting Amounts" of the asset accounts as "Unbalanced". 

**Describe the solution you'd like**
The problem is that these are assets that already existed before. This should be able to be specified as a "starting balance" when an account is created?

**Comment by powerpaul17** [comment]
Hello,

yes that's the current behavior as I haven't implemented a starting 
balance for accounts yet. The "Unbalanced" value is also not completely 
correct as it is only calculated from the account type summaries and 
not from the individual transactions.

**Comment by powerpaul17** [comment]
I'm also not sure if it's good to add a starting balance to an account or if it should be handled by a "starting balance account" under the equity section. I think Gnucash handles it this way..

**Comment by SafetyIng** [comment]
Traditionally in double-entry bookkeeping, there is an account for starting assets and ending assets for the respective accounting year. Perhaps it would be useful to automatically set up the accounts for an account system accordingly and to have the option of having them displayed or hidden in the graphics?

**Comment by powerpaul17** [comment]
That is more or less how Gnucash describes it. I think they have 'equity' accounts for that. In any case automatic generation of accounts is a bit problematic since it's impossible to cover all user's cases..

**Comment by frake65** [comment]
Probably the usual workaround (and also the one everybody will find theirselves) for this issue is to set a starting balance as first transaction for a time before the imported or "really" booked transactions start.

**Comment by powerpaul17** [comment]
That's right but it leaves the transaction unbalanced since there is no corresponding account. I am thinking about adding an equity account type but I'm not quite sure about the details.
